onfido/test/test_integration.py

```diff
@@ -57,6 +57,14 @@ def test_integration_document(self):
 
         self.assertIn("id", doc_response)
 
+    def test_integration_live_photo(self):
+        api = self.create_api()
+        applicant = random.choice(api.Applicants.all()["applicants"])
```

Comment:
Using random functions in tests? Don't we want them to be reproducible all the time?